src/chrome/content/rules/Massengeschmack.tv.xml

```diff
@@ -11,8 +11,13 @@
     <target host="alpha.massengeschmack.tv" />
     <target host="beta.massengeschmack.tv" />
 
-	<target host="forum.massengeschmack.tv" platform="mixedcontent"/>
-
+<!--
+    Mixed content:
+        - Images from schnittbrot.com *
+        - Images from www.schnittbrot.com *
```

Comment:
The host contains also mixed script, so We can't keep this target active without breaking things.